compose.yml

```diff
@@ -1,52 +1,12 @@
 services:
-  development:
+  fcp-mpdp-frontend:
```

Comment:
Would remove the `container_name` to avoid conflicts.  Let Docker decide the name based on running context